Source/Core/Core/AchievementManager.cpp

```diff
@@ -113,11 +114,9 @@ void AchievementManager::LoadGameByFilenameAsync(const std::string& iso_path,
           },
       .close = [](void* file_handle) { delete reinterpret_cast<FilereaderState*>(file_handle); }};
   rc_hash_init_custom_filereader(&volume_reader);
-  std::array<char, HASH_LENGTH> game_hash;
-  if (!rc_hash_generate_from_file(game_hash.data(), RC_CONSOLE_GAMECUBE, iso_path.c_str()))
-    return;
-  m_queue.EmplaceItem([this, callback, game_hash] {
-    const auto resolve_hash_response = ResolveHash(game_hash);
+  rc_hash_generate_from_file(m_game_hash.data(), RC_CONSOLE_GAMECUBE, iso_path.c_str());
```

Comment:
Why was the error check for this removed? As far as I know, this can still fail for filesystem reasons (e.g. consider a game on external storage being accidentally knocked loose/removed).